guides/runbooks/orca-zombie-executions.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: single
+title:  "Orca: Zombie Executions"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Aliases: `orphaned execution`
+
+A pipeline or task is "stuck" in a `RUNNING` state but not doing anything.
+
+# Diagnosis
+
+Logs will be emitted regularly for Executions that are currently running in Orca.
+If no logs have been emitted for over 10 minutes for a `RUNNING` Execution, it is very likely a zombie.
+
+## Metrics & Alerting
+
+When enabled, Orca can be configured to detect and emit metrics for zombie Executions. This setting is expensive with the `RedisExecutionRepository`.
+You can enable this detection by setting `queue.zombieCheck.enabled: true` in your Orca configuration.
+
+When enabled, any discovered zombies will be logged out, as well as emitted via a metric:
+
+```
+Found zombie executionType=pipeline application=myapplication executionName=myexample executionId=01CS076X85RX6MWBTQ0VGBF8VX
+```
+
+An alert should be set on the metric `queue.zombies`, triggering whenever there is a count greater than 0.
+
+# Remediation
+
+## Rehydrate Queue
+
+You can attempt to re-hydrate the queue using an [admin API in Orca](https://github.com/spinnaker/orca/blob/master/orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/admin/web/QueueAdminController.kt#L33), which must be called directly as it is not exposed through Gate.
```

Comment:
It's not clear here what rehydrating a queue is or what it does. Would be great if that was defined.

Reply:
Updated. I also moved a couple sentences from the `Known Causes` part up to the summary, which I think sets better context for the rest of the doc. WDYT?